src/App.tsx

```diff
@@ -17,18 +88,43 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                input={input}
+                status={status}
+                onChangeStatus={handleChangeStatus}
+                onChangeInput={handleChangeInput}
+                onClearInput={clearInput}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {data.length > 0
```

Comment:
```suggestion
              {!!data.length
```

Reply:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/80322190/b5ae7398-213d-41da-a5e1-bb8699ee7e05)

Reply:
I fix it this way => data.length